python/lsst/ctrl/bps/wms/htcondor/htcondor_service.py

```diff
@@ -599,18 +599,41 @@ def _replace_file_vars(config, arguments, workflow, gwjob):
 
     # Replace input file placeholders with paths.
     for gwfile in workflow.get_job_inputs(gwjob.name, data=True, transfer_only=False):
-        if gwfile.wms_transfer and not use_shared or not gwfile.job_shared:
-            uri = os.path.basename(gwfile.src_uri)
-        else:
+        if not gwfile.wms_transfer:
+            # Must assume full URI if in command line and told WMS is not
+            # responsible for transferring file.
             uri = gwfile.src_uri
+        elif use_shared:
+            if gwfile.job_shared:
+                # Have shared filesystems and jobs can share file.
+                uri = gwfile.src_uri
+            else:
+                # Taking advantage of inside knowledge.  Not future-proof.
+                # Temporary fix until have job wrapper that pulls files
+                # within job.
+                if use_shared and not gwfile.job_shared and gwfile.name == "butlerConfig" \
+                   and not Path(gwfile.src_uri).suffix:
```

Comment:
You may consider using parentheses for wrapping this condition instead of `\`; see [this](https://developer.lsst.io/python/style.html#python-s-implied-continuation-inside-parens-brackets-and-braces-should-be-used-for-wrapped-lines) example from the LSST Dev Guide. And regardless which method you will use, the continued line (615) must be distinguished from the following line through increased indentation.